### PR TITLE
Jseldess/basic sql statements

### PR DESCRIPTION
**Pull request by jseldess**
@spencerkimball, @petermattis, I've updated the sql examples on the **Basic SQL Statements** to be less generic. Context now is a `bank` database with an `accounts` table. Look ok? 

cc: @mjibson

### CODE REVIEW

File: basic-sql-statements.md [hunk 1]
```diff
@@ -122,18 +116,18 @@ To see all tables in the active database, use the `SHOW TABLES` statement:
...
+| accounts |
+| users    |
++----------+
 ~~~
 
 To view tables in a database that's not active, use `SHOW TABLES FROM` followed by the name of the database:
 
 ~~~ sql
-SHOW TABLES FROM db2;
+SHOW TABLES FROM animals;
```
Comment (mjibson) [inline basic-sql-statements.md]: animals?

File: basic-sql-statements.md [hunk 2]
```diff
@@ -62,20 +62,18 @@ SHOW DATABASE;
...
 +----------+
-| db1      |
+| bank     |
 +----------+
 ~~~
 
 ## Create a Table
 
-To create a table, use the `CREATE TABLE` statement followed by a table name, the column names, the data type for each column, and the primary key constraint:
+To create a table, use the `CREATE TABLE` statement followed by a table name, the column names,  and the data type and constraint, if any, for each column:
```
Comment (mjibson) [inline basic-sql-statements.md]: two spaces near `column names,  and`

File: basic-sql-statements.md [hunk 3]
```diff
@@ -153,39 +147,39 @@ SHOW TABLES FROM db2;
...
 ~~~
 
-[Defaults values](default-values.html) are used when you leave specific columns out of your statement, or when you explicitly request default values. For example, either of the following statements would create a row with `column_b` filled with its default value, in this case `NULL`:
+[Defaults values](default-values.html) are used when you leave specific columns out of your statement, or when you explicitly request default values. For example, either of the following statements would create a row with `balance` filled with its default value, in this case `NULL`:
 
 ~~~ sql
-INSERT INTO table1 (column_d, column_c, column_a) VALUES 
-    ('aaaa', DATE '2016-01-01', 12345);
+INSERT INTO accounts (id, balance) VALUES 
+    (1);
```
Comment (mjibson) [inline basic-sql-statements.md]: This 1 needs to be a 3 if the previous inserts are run because id is a PK.

File: basic-sql-statements.md [hunk 4]
```diff
@@ -153,39 +147,39 @@ SHOW TABLES FROM db2;
...
 
-INSERT INTO table1 (column_d, column_c, column_b, column_a) VALUES 
-    ('aaaa', DATE '2016-01-01', DEFAULT, 12345);
+INSERT INTO accounts (id, balance) VALUES 
+    (1, DEFAULT);
```
Comment (mjibson) [inline basic-sql-statements.md]: This 1 needs to be a 4 if the previous inserts are run because id is a PK.

File: basic-sql-statements.md (hunk 2 above)
Comment (jseldess) [inline basic-sql-statements.md]: thanks, fixed with my next commit.

File: basic-sql-statements.md (hunk 1 above)
Comment (jseldess) [inline basic-sql-statements.md]: Haha, yeah. A bit of silliness ;). But really, I'm happy to change that second database to whatever. What do you think makes sense as a second database in the context of a bank?

Comment (mjibson) [inline basic-sql-statements.md]: Oh I thought it was a reference to a database created in this document. No problem with the silly name, just thought it was a typo.

File: basic-sql-statements.md (hunk 3 above)
Comment (jseldess) [inline basic-sql-statements.md]: Updated examples to be a sequence of inserts.

**Comment by mjibson** [comment]
LGTM

**Pull request merged**